modules/nw-sriov-cfg-bond-interface-with-virtual-functions.adoc

```diff
@@ -9,7 +9,7 @@ Bonding enables multiple network interfaces to be aggregated into a single logic
 
 Bond-CNI can be created using SR-IOV virtual functions and placing them in the container network namespace.
 
-{product-title} only supports Bond-CNI using SR-IOV virtual functions. Physical interfaces are not supported.
+{product-title} only supports Bond-CNI using SR-IOV virtual functions, managed by SR-IOV CNI. Other CNI or types of interfaces are not supported.
```

Comment:
You could provide the full name of the "SR-IOV" and "CNI" acronyms on their first use in this module. At present, neither of those acronyms are named in full in this module.